sdks/python/test-suites/dataflow/build.gradle

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
why do we need build.gradle in addition to common.grade?

Reply:
Nevermind, I see that they both serve a different purpose.

Reply:
It seems that we need to make changes to the top-level suite definitions, e.g. : https://github.com/apache/beam/blob/ba380270dbc8854ad44f6636a6767317a0416a99/build.gradle#L216
to point to /test-suites/<runner>/build.gradle files.

Reply:
I've rollbacked too much.
Made top level build.gradle tasks depend on test-suites ones AMAP.